actions/dhcp/README.md

```diff
@@ -0,0 +1,9 @@
+---
+slug: Re DHCP
+name: Re DHCP
+tags: disk
```

Comment:
Tags are important because ArtifactHub uses them to cross-reference between projects. if we use them wisely we will be able to have a correlation with Falco rules or Helm charts that are related to the same topic.

Now sure if it will turn out to be useful but `disk` looks wrong. You can use `dhcp` and `networking`